GitExtensions.Extensibility/GitExtensions.Extensibility.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We already have the https://github.com/gitextensions/gitextensions.extensibility, and should not be creating another project in the main solution. That project should contain all base interfaces and extensibility points.
The main solution as well as all Git Extensions plugins must depend on that project and install it as a NuGet dependency.

Reply:
This project to migrate to https://github.com/gitextensions/gitextensions.extensibility.
Do you think this should be done right away?

Reply:
I'd rather move it, then create noise in the history.